models/users.py

```diff
@@ -13,8 +13,7 @@
 Once more welcome aboard ship Nebuchadnezzar in the city of zion (^__^).
 """
 
-from sqlalchemy import (Column, Date,
-    Integer, String, Date, ForeignKey, Text)
+from sqlalchemy import (Column, Integer, String, Date, ForeignKey, Text)
```

Comment:
You can use brackets if you have imports formatted on the more than 1line 
Otherwise on same line a comma is okay